### PR TITLE
Improve chinese_simplified localization

### PR DESCRIPTION
**Pull request by violarulan**
In the Chinese localization file, there are some missing translations, resulting in several English words appearing.

This PR brings the localization to where it should be and standardizes existing translation phrases.

### CODE REVIEW

File: application/language/chinese_simplified/filter_lang.php [hunk 1]
```diff
@@ -105,21 +105,21 @@
...
+$lang['filter_actions_recv_direct'] = '已接收(直邮)';
+$lang['filter_actions_recv_electronic'] = '已接收(电子)';
 $lang['filter_actions_not_rcvd'] = "Not Received";
 $lang['filter_actions_create_adif'] = '创建 ADIF';
 $lang['filter_actions_print_label'] = '打印标签';
 $lang['filter_actions_start_print_title'] = '打印标签';
 $lang['filter_actions_print_include_via'] = "包含通过";
 $lang['filter_actions_print_include_grid'] = '包含网格？';
-$lang['filter_actions_print_include_ref'] = "Include reference? (SIG, SOTA, POTA, IOTA, WWFF; If available in location)";
+$lang['filter_actions_print_include_ref'] = "Include reference?(SIG, SOTA, POTA, IOTA, WWFF; If available in location)";
```
Comment (HB9HIL) [inline application/language/chinese_simplified/filter_lang.php:122]: Can you translate this line aswell? :-)

**Comment by HB9HIL** [comment]
Hello @violarulan 

Thank you very much for this improvement :)

Can you translate the marked line aswell to make it complete?

**Comment by HB9HIL** [comment]
Additionally I noticed you based your PR against `master` branch. I switched it over to `dev` branch. So you can see there would be much more

**Comment by violarulan** [comment]
> Additionally I noticed you based your PR against `master` branch. I switched it over to `dev` branch. So you can see there would be much more
> 
> perform `git pull` before adding a new commit

Sure thing and thanks for the correction. I've added more translation. It should be completed by now.

_...and please ignore my misuse of issue referencing in commit message._

**Review by HB9HIL — APPROVED** [review]
Thank you very much! Contributing Language is really important. We appreciate it :)

**Comment by HB9HIL** [comment]
Hello @violarulan 

We switched over to `gettext()` and po files. Now much more chinese translations are waiting for translations :)

Check this here: https://translate.wavelog.org/engage/wavelog/

**Comment by violarulan** [comment]
Thanks @HB9HIL

That's a good translation platform. I've registered an account. I'll do the translation work there.

**Pull request merged**